3/두개뽑아서더하기.js

```diff
@@ -0,0 +1,13 @@
+function solution(numbers) {
+  const answer = [];
+
+  for (let i = 0; i < numbers.length - 1; i++) {
+    for (let j = i + 1; j < numbers.length; j++) {
+      answer.push(numbers[i] + numbers[j]);
+    }
+  }
+
+  const result = [...new Set(answer)];
+
+  return result.sort((a, b) => a - b);
```

Comment:
```suggestion
  return [...new Set(answer)].sort((a, b) => a - b);
```
한 번에 입력 받아서 return할 수 있어요.